lib/scrimp/app.rb

```diff
@@ -55,6 +55,19 @@ class App < Sinatra::Base
       protocols.to_json
     end
 
+    get '/transports' do
+      transports = {'Thrift::BufferedTransport'=>'Thrift::BufferedTransport',
+                    'Thrift::FramedTransport'=>'Thrift::FramedTransport'}
+      content_type :json
+      transports.to_json
+    end
+
+    get '/sockets' do
+      sockets = {'Thrift::Socket'=>'Thrift::Socket'}
```

Comment:
Are there other socket implementations? I'd like to understand the motivation for making this configurable.